src/components/orderDetails.js

```diff
@@ -10,12 +10,56 @@ function prettyDate(rawDate) {
     return humanDate;
   }
 
+  class ExchangeTotal extends React.Component {
+    constructor(props) {
+        super(props)
+    }
+    render () {
+        return (
+            <div>
+                <p>{this.props.count}</p>
+            </div>
+        )
+    }
+}
+
+
+class RefundTotal extends React.Component {
+  constructor(props) {
+      super(props)
+  }
+  calRefundTotal(){
+    const qty = this.props.qty;
+    const price = 100;
+    const result = price * qty;
+    return result;
+  }
```

Comment:
Good for now but will need updating in future -- as it is currently set out, this function assumes that all items have the same price.

One approach you can take is to get rid of the `qty` prop, and replace it with an `returnDetails` prop (or something similar), where you plug in the return details information from the parent component. (E.g., `<RefundTotal return-details={this.state.returnDetails} />`).

Then, in this component, `calRefundTotal()` can look at each of the lines in the `returnDetails` prop, multiply the quantity by the price, and figure out the grand total. For example:

```js
calRefundTotal() {
  let grandTotal = 0;
  this.props.returnDetails.items.map(item => {
    grandTotal = grandTotal + (item.qty * item.price);
  })
  return grandTotal;
}
```

You don't need to update this for this commit -- this note is here for the future.